src/views/open-api-docs-view-v2/components/operation-content/index.tsx

```diff
@@ -47,17 +48,22 @@ export interface OperationProps {
  * Render detailed content for an individual operation.
  */
 export default function OperationContent({
```

Comment:
Similar to landing content, the changes in this file add support for a `versionSwitcherSlot` in `OperationContent`.